azure-iot-device/tests/common/pipeline_test.py

```diff
@@ -0,0 +1,117 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
```

Comment:
This file, if I understand it, contains helper functions for use in various pipeline tests. So I have two follow up questions:

1) Could we perhaps name it pipeline_test_helpers.py instead to make it clearer?
2) Why is this located in the root of the tests.common directory? Shouldn't this be in tests.common.pipeline?

Reply:
Will revisit in a different PR.  I agree that this is awkward.  

---
In reply to: [287540855](https://github.com/Azure/azure-iot-sdk-python-preview/pull/94#discussion_r287540855) [](ancestors = 287540855)